backends/graphs/resources/firewall.fullGraph.json

```diff
@@ -0,0 +1,496 @@
+{
+  "name" : "program",
+  "nodes" : [
+    {
+      "type" : "parser",
+      "name" : "MyParser",
+      "nodes" : [
+        {
+          "node_nmb" : 0,
+          "name" : "start\n(hdr.ethernet.etherType)",
+          "type" : "state",
+          "type_enum" : 8
+        },
+        {
+          "node_nmb" : 1,
+          "name" : "parse_ipv4\n(hdr.ipv4.protocol)",
+          "type" : "state",
+          "type_enum" : 8
+        },
+        {
+          "node_nmb" : 2,
+          "name" : "tcp",
+          "type" : "state",
+          "type_enum" : 8
+        },
+        {
+          "node_nmb" : 3,
+          "name" : "accept",
+          "type" : "state",
+          "type_enum" : 8
+        },
+        {
+          "node_nmb" : 4,
+          "name" : "reject",
+          "type" : "state",
+          "type_enum" : 8
+        }
+      ],
+      "transitions" : [
+        {
+          "from" : 0,
+          "to" : 1,
+          "cond" : ""
```

Comment:
why have all these empty cond fields?

Reply:
When transition between nodes has no label, 'cond' field is empty (has value "").

If that is a problem, I could remove addition of this entry when value is "".

Reply:
Issue fixed